cinema_service/settings.py

```diff
@@ -116,6 +116,15 @@
 
 AUTH_USER_MODEL = "user.User"
 
+# REST_FRAMEWORK = {
+#     "DEFAULT_PERMISSION_CLASSES": [
+#         "cinema.permissions.IsAdminOrIfAuthenticatedReadOnly"
+#     ],
+#     "DEFAULT_AUTHENTICATION_CLASSES": [
+#         "rest_framework.authentication.TokenAuthentication"
+#     ],
+# }
+
```

Comment:
use it to avoid duplication

Reply:
I added default permission classes and removed duplication, but now I have some errors and I do not understand how to fix them